app/components/track-item/template.hbs

```diff
@@ -1,12 +1,39 @@
-{{#link-to 'channel.tracks.track.index' track class="Track-link"}}
-	<h3 class="Track-title">
-		{{track.title}}
-	</h3>
-	<p class="Track-body">{{track.body}}</p>
-{{/link-to}}
+{{#if inline}}
+	<a href="#" {{action "play" track}}>
+		<h3 class="Track-title">
+			{{track.title}}
+		</h3>
+		<p class="Track-body">{{track.body}}</p>
+	</a>
+{{else}}
+	{{#link-to 'channel.tracks.track.index' track class="Track-link"}}
+		<h3 class="Track-title">
+			{{track.title}}
+		</h3>
+		<p class="Track-body">{{track.body}}</p>
+	{{/link-to}}
+{{/if}}
 
 {{#if track.channel.canEdit}}
```

Comment:
if we move to a [...] contextual menu, we'll be able to avoid this if; the menu will always be showed, but when clicked it can render what actions are available to the current user depending its auth status

Reply:
Isn't this a different action? One is when you tap the track itself, another is the (future) contextual menu.